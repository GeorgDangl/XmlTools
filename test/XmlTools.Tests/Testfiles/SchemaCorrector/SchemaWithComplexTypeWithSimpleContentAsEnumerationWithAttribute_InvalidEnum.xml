<?xml version="1.0" encoding="UTF-8"?>
<Monsters>
  <Monster Resistance="Fire">Troll</Monster>
  <Monster Resistance="Poison">Orc</Monster>
  <Monster Resistance="Earth">Ogre</Monster>
  <Monster Resistance="Ice">Lich</Monster>
</Monsters>