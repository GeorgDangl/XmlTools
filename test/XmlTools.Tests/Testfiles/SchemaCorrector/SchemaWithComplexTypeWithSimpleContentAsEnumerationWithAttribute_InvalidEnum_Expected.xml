<?xml version="1.0" encoding="UTF-8"?>
<Monsters>
  <Monster Resistance="Poison">Orc</Monster>
  <Monster>Ogre</Monster>
  <Monster Resistance="Ice">Lich</Monster>
</Monsters>